app/react/src/server/config/babel.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
         targets: {
           browsers: ['last 2 versions', 'safari >= 7'],
         },
-        modules: false,
+        modules: process.env.NODE_ENV === 'test' ? undefined : false,
```

Comment:
not sure if we should do `undefined` here or `commonjs`. the default for this is `commonjs` but I'm worried it might change in future versions and break support.